src/core/search/rax_tree_test.cc

```diff
@@ -74,6 +75,12 @@ TEST_F(RaxTreeTest, LowerBound) {
 
   EXPECT_TRUE(it1 == map.end());
   EXPECT_TRUE(it2 == keys.end());
+
+  // Test lower bound empty string
+  vector<string> keys2;
+  for (auto it = map.lower_bound(""); it != map.end(); ++it)
```

Comment:
it's not enough. 
`""` actually is non-null string_view. I am talking about `string_view{}`.
Also the same with find and insert (empty string)